bls-signatures/src/hash.rs

```diff
@@ -6,7 +6,7 @@ use {
 /// Domain separation tag used for hashing messages to curve points to prevent
 /// potential conflicts between different BLS implementations. This is defined
 /// as the ciphersuite ID string as recommended in the standard
-/// https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-bls-signature-05#section-4.2.1.
+/// <https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-bls-signature-05#section-4.2.1>.
```

Comment:
Maybe we can change this and the one below in bls-signatures to a named link as well for consistency, but I can probably just take care of that on a next PR on BLS.

Reply:
I had to rebase anyway, so I made the change, let me know what you think!